accepted/0000-hub-online-synchronization.md

```diff
@@ -136,6 +136,29 @@ This is needed when we want to make this API organization independent.
 In the existing API the calling user is only allowed to operate on his own organization any maybe utilize "Vendor Channel" when he is a "SUSE Manager Administrator".
 
 
+#### The Peripheral Server system entry
+
+Similar as in the Proxy case, we should have a "system" entry which represent the Peripheral Server.
+This can be used to show special information or relations.
+
+There are 2 different cases we need to consider.
+   1. A host system exists with the same FQDN (podman running the container)
+   2. No host system got registered (kubernetes case or podman where the host should not be registered at the Hub)
+
+To handle these 2 cases, we should create a new base entitlement for "Container Workloads" (similar to Foreign Entitlement) and 2 add-on entitlements for Peripheral and Proxy Server.
+In case we have a host system, we would not use the "Container Workload Entitlement" as base, but the "Salt Entitlement".
+Only in case if no host is available, the "Container Workload Entitlement" would be used.
```

Comment:
Every 5.0 suse manger instance currently has same static hostname  `uyuni-server.mgr.internal`, how are we we gonna differentiate between all these peripheral servers.

Reply:
This is the "internal" hostname in the container network. We need the hostname from external. Otherwise the hub cannot connect as the internal name is not routed.

Reply:
Got it! I just wanted to highlight that the internal hostname doesn’t become an issue.